singularity-eos/eos/eos_spiner.cpp

```diff
@@ -842,7 +842,8 @@ Real SpinerEOSDependsRhoT::lTFromlRhoSie_(const Real lRho,
   if ( whereAmI == TableStatus::OffBottom ) {
     // on the cold curve. No unique temperature.
     // return the temperature at the cold curve crossing.
-    lT = lTColdCrit_.interpToReal(lRho);
+    // lT = lTColdCrit_.interpToReal(lRho);
```

Comment:
the comments here probably need to change and we should remove the commented out line of code if this is the final version for the PR.

Reply:
Done